File: src/Net/Store/Before/Customer.cs
/*
 * Copyright (C) 2010-2011 by ExampleWorks, Inc. All rights reserved.
 * LICENSE: You can't redistribute it and/or modify it.
 */
namespace Before
{
    public class Customer : ThirdParty
    {
        public string FirstName { get; private set; }

        public string LastName { get; private set; }

        public Customer(string firstName, string lastName, string phoneNumber)
            : base(phoneNumber)
        {
            this.FirstName = firstName;
            this.LastName = lastName;
        }
    }
}

File: src/Java/Store/4. Third Hole/src/main/refactoringgolf/store/Customer.java
package refactoringgolf.store;

public class Customer extends ThirdParty {
	
    public String firstName;
    public String fastName;

    public Customer(String firstName, String lastName, String phoneNumber)
    {
    	super(phoneNumber);
        this.firstName = firstName;
        this.fastName = lastName;
    }

	public String getFirstName() {
		return firstName;
	}

	public String getFastName() {
		return fastName;
	}
}
